tests/integration/test_watchman_make.py

```diff
@@ -0,0 +1,114 @@
+# vim: ts=4:sw=4:sts=4:et:
+# Copyright 2012-present Facebook, Inc.
+# Licensed under the Apache License, Version 2.0
+
+from __future__ import absolute_import
+from __future__ import division
+from __future__ import print_function
+# no unicode literals
+
+import WatchmanTestCase
+import os
+import os.path
+import sys
+import re
+import subprocess
+import signal
+import time
+import uuid
+
+class TestWatchmanMake(WatchmanTestCase.WatchmanTestCase):
+
+    def setUp(self):
+        if sys.platform == 'win32':
+            self.skipTest('Continuous integration testing only in Linux / MacOS')
```

Comment:
Oh derp, I forgot about `/bin/echo`... but no, it is because I can't sigint on windows, and I have to send a subprocess.CTRL_C_EVENT, which is available only on windows. I had a commit with that, but it still failed, so I decided to run the tests only under Unixes, where subprocess acts nicely. In the end, watchman-make will still work on Windows, it's only the testing (spawning processes) which is much easier on *nix. :-)